packages/components/src/spectrum-deephaven-darkest.module.scss

```diff
@@ -394,6 +395,12 @@
   --spectrum-purple-visual-color: var(--spectrum-purple-900);
   --spectrum-fuchsia-visual-color: var(--spectrum-fuchsia-900);
   --spectrum-magenta-visual-color: var(--spectrum-magenta-900);
+
+  // NEW specrum variables to enable RGBA usage
+  --spectrum-gray-800-rgb: 240, 240, 238;
```

Comment:
If these variables don't actually originate from Spectrum, what's the reason to give them the --spectrum prefix?